.gitignore

```diff
@@ -1,3 +1,3 @@
 /*/target/
 /target/
-core/src/main/java/com/threerings/getdown/data/Build.java
+core/src/main/java/com/threerings/getdown/data/Build.java
```

Comment:
the file is still changed. looks like trailing endline was lost

Reply:
In fact, I see there no difference. There is no linefeed here and there. :(